source/release-notes.txt

```diff
@@ -10,6 +10,44 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.40.0
+----------------
+
+*Released September 26, 2023*
+
+New Features:
+
+- Upgrade :ref:`embedded MongoDB shell <embedded-mongodb-shell>` to
+  version 2.0.0 (:issue:`COMPASS-7057`).
+  
+- Upgrade Node driver to version 6.0.0 (:issue:`COMPASS-7057`).
+
+- Upgrade Electron to version 26.
+
+- Set up local Atlas detection (:issue:`COMPASS-7213`).
+
+- Display local Atlas development environments as such
+  (:issue:`COMPASS-7156`).
+
+- Introduce ``$vectorSearch`` aggregation stage to MongoDB 7.1 and 7.0.x
+  (:issue:`COMPASS-7064`).
+
+- Enable `Atlas search index
+  <https://www.mongodb.com/docs/atlas/atlas-search/atlas-search-overview/#fts-indexes>`__
+  management (:issue:`COMPASS-7238`).
+
```

Comment:
Let's add an item for the generative ai feature?
```suggestion
- Enable `Atlas search index
  <https://www.mongodb.com/docs/atlas/atlas-search/atlas-search-overview/#fts-indexes>`__
  management (:issue:`COMPASS-7238`).

- Enable natural language query and pipeline generation (incrementally rolled out to users) (:issue:`COMPASS-6866`)

```

Reply:
I wasn't sure if we wanted to include this since I think I saw that it wasn't fully rolled out yet? I'll add this in though!